jep/6/README.adoc

```diff
@@ -0,0 +1,159 @@
+
+. To be able to submit data to CVE, every individual involved in providing that data (i.e. the authors of the CVE description, not the reporter(s) of the vulnerability, not the developer(s) fixing it, not the author(s) of the Jenkins security advisory) will need to agree to the https://cve.mitre.org/about/termsofuse.html[MITRE CVE Terms of service].
+. As a CNA operating under the DWF project, the Jenkins project will need to agree to the http://contributor-covenant.org/version/1/4/[DWF Contributor Agreement], which is based on Contributor Covenant 1.4.
+. The Jenkins project will need to agree to the http://cve.mitre.org/cve/cna/rules.html[MITRE CNA rules] that outlines the rules and processes the Jenkins project will need to follow and implement. The following sections apply to the Jenkins project:
+.. Chapter 2 outlines the responsibilities of all CNA, which applies to us.
+.. Chapter 4 outlines the CNA candidate process that includes onboarding.
+
+The Distributed Weakness Filing Project splits the CNA responsibilities into two parts:
+
+* The Jenkins Project, as CNA, will own blocks of CVE numbers.
+* Individuals working with the Jenkins Project, the CVE mentors, will assign CVE numbers and write issue descriptions in the required format.
+
+To start this process, an individual associated with the Jenkins project will need to become a CVE mentor.
+Once they are a CVE mentor, the Jenkins project can apply to become a CNA.
+
+
+== Motivation
+
+CVE, or Common Vulnerabilities and Exposures, is a system to unambiguously identify security vulnerabilities in software. Quoting link:https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures#CVE_Usage[Wikipedia]:
+
+____
+Common Vulnerabilities and Exposures (CVE®) is a dictionary of common names (i.e., CVE Identifiers) for publicly known information security vulnerabilities. CVE’s common identifiers make it easier to share data across separate network security databases and tools, and provide a baseline for evaluating the coverage of an organization’s security tools. If a report from one of your security tools incorporates CVE Identifiers, you may then quickly and accurately access fix information in one or more separate CVE-compatible databases to remediate the problem.
+____
+
+The Jenkins project has used CVE Identifiers for security vulnerabilities since 2013, after Kurt Seifried of Red Hat, Inc. requested we do so.
+Since then we've been providing advance information about soon-to-be-fixed security vulnerabilities to the private link:http://oss-security.openwall.org/wiki/mailing-lists/distros[distros mailing list] whose members are various Linux and BSD distributions.
+Most of those CVEs were assigned by Kurt Seifried.
+In early 2017, the list owner requested that these CVE assignments be handled elsewhere due to the lack of relevant of advance information about Jenkins vulnerabilities to the regular list recipients.
```

Comment:
I don't understand the phrasing of this sentence.  I think it might mean one of the following two things (or something else entirely):

> In early 2017, the list owner requested that these CVE assignments be handled elsewhere because regular list recipients do not have access to advance information about Jenkins vulnerabilities.

or 

> In early 2017, the list owner requested that these CVE assignments be handled elsewhere because Jenkins vulnerabilities were not relevant to regular list recipients.

Can you clarify which of those is intended?  Alternately, since this is the rationale, the phrasing may not matter.